.ci/jenkinsfile_hub_uploader.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline{
+    agent {
+        docker {
+            // if the docker file changes it should be build and uploaded manually
+            image 'harbor.mellanox.com/swx-storage/ci-demo/x86_64/swx-dockerhub-uploader:20240221'
+            label 'DOCKER'
+            args '-v /auto/mswg:/auto/mswg -v /mswg:/mswg -v /var/run/docker.sock:/var/run/docker.sock --privileged'
```

Comment:
do you need full /auto/mswg access? I suggest adding only /auto/mswg/release/ufm/plugins